cmd/issuer-rest/static/creditscore.html

```diff
@@ -76,12 +80,7 @@ <h5 class="w-full my-2 text-xl leading-tight left text-black">Find the services
             <div class="flex-none mt-auto bg-white rounded-b rounded-t-none overflow-hidden shadow p-6">
                 <img class="object-contain h-64 w-full" src="img/credit_score.jpg">
                 <div class="flex justify-center mt-8 pt-2">
-                    <form id="didCommDemo"  action="">
-                        <input type="text" id="demoType" name="demoType" value="" checked hidden>
-                        <input type="text" id="didCommScope" name="didCommScope" value="" checked hidden>
-                        <input type="text" id="adapterProfile" name="adapterProfile" value="" checked hidden>
-                        <input type="submit" id="creditScore" class="lg:mx-0 hover:underline credit-gradient text-white font-bold my-6 py-4 px-8 shadow-lg" value="Get your credit report">
-                    </form>
+                    <input type="submit" id="getDrivingLicense" onclick="getDrivingLicense()" class="lg:mx-0 hover:underline credit-gradient text-white font-bold my-6 py-4 px-8 shadow-lg" value="Get your credit report">
```

Comment:
Just wondering if we should keep the existing `credit-score` issuer as is (`New Bank Account` use case) and create a new `credit-score` issuer with DL creds (`Government RP` use case).

Reply:
done add new page creditscorenologin.html